src/Settings/SelectInventoryList.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+import { SafeAreaView } from 'react-navigation';
+import Meteor, { Accounts } from 'react-native-meteor';
+import TableView from 'react-native-tableview'
+const { Section, Item } = TableView;
+
+export default class SelectInventoryList extends Component {
+
+	constructor(props) {
+		super(props);
+
+		const defaultLists = this.props.screenProps.inventoryLists.filter(list => list.isDefault);
+
+		let defaultList;
+		if (defaultLists.length < 1) {
```

Comment:
This technically shouldn't be a possible scenario, right? I think we talked about either creating a default inventory list to begin with, or automatically making the first inventory list the default?

Reply:
Right this is just in case...